docs/community/faqs.md

```diff
@@ -1,34 +1,42 @@
+
+   **Additional resources**:
+
+   - ["Who's looking for open source contributors?"](https://dev.to/opensauced/whos-looking-for-open-source-contributors-week-46-247i)
+   - [Intro to Open Source Course](https://intro.opensauced.pizza/)
+
 2. **My contribution does not show up on my OpenSauced profile. How do I fix it?**
-Create an [insight page](https://app.opensauced.pizza/hub/insights) or add your merged pull request to [a highlight](https://app.opensauced.pizza/feed).
+
+   Create an [insight page](https://app.opensauced.pizza/hub/insights) or add your merged pull request to [a highlight](https://app.opensauced.pizza/feed).
 
 3. **I want to provide feedback on OpenSauced.**
-We love feedback. Post your suggestion in our [feedback repository](https://github.com/orgs/open-sauced/discussions).
+
+   We love feedback. Post your suggestion in our [feedback repository](https://github.com/orgs/open-sauced/discussions).
 
 ## [Sync your GitHub Team](#sync-your-github-team)
-1. **I don't see my Organization on Sync your GitHub Team?**
-You may not see your Organization if you are not a "public" member. You can update this by going to the organization on GitHub and selecting the People tab. Next, search for your name. You can then change "private" to "public." For full instructions, see [this guide](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership).
 
-2. **I get an error when I try to select my team in Sync your GitHub Team**
-If you see your Organization, but you get an error when you try to select a team, your organization likely has third-party applications restricted. You can check this by going to your organization on GitHub, selecting Settings, and then Third-party access. 
+1. **I don't see my organization on "Sync your GitHub Team".**
+
+   You may not see your organization if you are not a "public" member. You can update this by going to the organization on GitHub and selecting the "People" tab. Next, search for your name. You can then change "private" to "public." For full instructions, see [this guide](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership).
+
+2. **I get an error when I try to select my team in "Sync your GitHub Team".**
+
+   If you see your organization but get an error when you try to select a team, your organization likely has restricted third-party applications. You can check this by going to "Your organizations" on GitHub, selecting "Settings", and then "Third-party Access".
```

Comment:
I can't check this out because I don't have access to any organizations. But if this instruction is the same with [this resource on GitHub](https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization), we might as well want to adjust it for clarification. Currently, it's not too clear where we can find "Your organization" on GitHub.

```suggestion
   If you see your organization but get an error when you try to select a team, your organization likely has restricted third-party applications. You can check this by going to your profile picture in the upper-right corner of GitHub and selecting "Your organizations". Then select "Settings", followed by selecting "Third-party Access".
```